posts/2022/sparse-PFT-gridding.ipynb

```diff
@@ -0,0 +1,12558 @@
+{
```

Comment:
<p>It would be cool if the PFT name showed up on this plot (and the others in this notebook). Unfortunately the information on pftname is not preserved in the history file.</p><p>If you want, you could pull this from the corresponding CLM parameter file. I found this from the global attributes of this netcdf file: PFT_physiological_constants_dataset = "clm5_params.c190529.nc".</p><p>I think this is the file location: /glade/p/cesm/cseg/inputdata/lnd/clm2/paramdata/clm5_params.c190529.nc</p><p>And you can use the variable 'pftname' from that file to get the appropriate string for type=1.</p><p>(This is completely optional, but potentially a nice labeling feature.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Ah I added it but the automatic labelling is not so useful :)</p>